src/name_bazaar/server/dev.cljs

```diff
@@ -73,6 +73,7 @@
                             :endpoints {:port 6200
                                         :middlewares [logging-middlewares]}
                             :web3 {:port 8549}
+                            :db {:opts {:memory true}}
```

Comment:
It shows you a bit about how to configure the dev server sqlite database. If you choose `:memory false` a `db.db` file gets created in current folder. I found it the easiest way to inspect server db in debugging, since you can't have two connections to an in memory sqlite database.